sdk/storage/inc/datalake/response_models.hpp

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "blobs/internal/protocol/blob_rest_client.hpp"
+#include "protocol/datalake_rest_client.hpp"
+
+namespace Azure { namespace Storage { namespace Files { namespace DataLake {
+
+  // ServiceClient models:
+
+  using ListFileSystemsResult = ServiceListFileSystemsResponse;
+
+  // FileSystemClient models:
```

Comment:
what's this?

Reply:
Some delimiter that I think useful for code grouping.